### PR TITLE
Organize the type of emotes in the emote menu

### PR DESCRIPTION
**Pull request by RobLaughlin**
This makes it a little easier to find the emote that you want if you are going to use the emote menu.

### CODE REVIEW

**Comment by kenany** [comment]
Fails linting:

``` sh
$ npm run lint

> betterdgg@1.3.7 lint betterdgg
> eslint betterdgg/ chrome/inject.js firefox/lib/ safari/


betterdgg/betterdgg/modules/emoticons.js
    7:61  error  Unexpected trailing comma  comma-dangle
  219:68  error  Unnecessary semicolon      no-extra-semi

✖ 2 problems (2 errors, 0 warnings)
```

**Comment by RobLaughlin** [comment]
Should be fine now.

File: betterdgg/modules/emoticons.js [hunk 1]
```diff
@@ -179,6 +188,40 @@
...
+                    // Remove old emote
+                    $(elem).parent().remove();
+                    
+                    // Filter emotes into their respective containers
+                    if ($.inArray($emoteName, EMOTICONS) > -1) { $('#DGGEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, TWITCH_EMOTICONS) > -1) { $('#TwitchEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, BTTV_EMOTICONS) > -1) { $('#BTTVEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, BBDGG_EMOTICONS) > -1) { $('#BBDGGEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, ANIMATED) > -1) { $('#BBDGGEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, SUBONLY) > -1) { null; }
```
Comment (kenany) [inline betterdgg/modules/emoticons.js:219]: I think you can just leave this blank like so:

``` js
else if ($.inArray($emoteName, SUBONLY) > -1) {}
```

Comment (kenany) [inline betterdgg/modules/emoticons.js:219]: Any reason not to have a section for the sub emotes?

Comment (RobLaughlin) [inline betterdgg/modules/emoticons.js:219]: I would have preferred leaving the block empty but JS Lint throws me this  

<pre>
[16:51:41] Starting 'lint'...
[16:51:42] 'lint' errored after 820 ms
[16:51:42] ESLintError in plugin 'gulp-eslint'
Message:
    Empty block statement.
Details:
    fileName: G:\Projects\Better Better Destiny GG\Fork\betterdgg\betterdgg\modules\emoticons.js
    lineNumber: 219
</pre>


So I just put null in just to make JS Lint happy. The sub emotes are already in their own container at the bottom of the emote menu, so I just decided to leave them there and delete the duplicate ones.

**Comment by downthecrop** [comment]
Tested, looks great! Thanks :+1:

File: betterdgg/modules/emoticons.js (hunk 1 above)
Comment (kenany) [inline betterdgg/modules/emoticons.js:219]: Oh, I didn't know that would fail linting. I think in retrospect we could have used `return;`?

**Pull request merged**